css/src/components/layout.scss

```diff
@@ -3,8 +3,8 @@ $side-bar-width-wide: 4fr !default;
 .layout {
 	display: flex;
 	flex-direction: column;
-	min-height: 100vh;
-	overflow-x: hidden;
+	min-inline-size: 100vh;
+	overflow-inline: hidden;
```

Comment:
This change removes vertical scrolling on the page so I can no longer view the entire page.

Reply:
You're right! Luckily changes addressing @BenDMyers's comments fixed this. Give it another look